e2e_tests/tests/experiment/test_tf_estimator.py

```diff
@@ -150,6 +150,28 @@ def test_mnist_estimator_data_layer_lfs(tf2: bool) -> None:
     run_mnist_estimator_data_layer_test(tf2, "lfs")
 
 
+@pytest.mark.parallel  # type: ignore
+@pytest.mark.parametrize("tf2", [True, False])  # type: ignore
+def test_custom_reducer_distributed(secrets: Dict[str, str], tf2: bool) -> None:
+    config = conf.load_config(conf.fixtures_path("estimator_dataset/distributed.yaml"))
+    # Run with multiple steps to verify we are resetting reducers right.
+    config = conf.set_max_steps(config, 2)
+    config = conf.set_slots_per_trial(config, 2)
```

Comment:
blocking: this experiment will never get scheduled since this needs to be 8 slots.

Reply:
good catch.  I was testing on 2 GPUs, and I thought I updated it back to 8, but I guess not.